templates/content/blank/NewApp.Android/NewApp.Android.fsproj

```diff
@@ -210,5 +210,4 @@
   <Import Project="$(SolutionDir)/packages/Xamarin.Android.Support.Design.XamarinAndroidSdkVersion/build/MonoAndroid81/Xamarin.Android.Support.Design.targets" Condition="Exists('$(SolutionDir)/packages/Xamarin.Android.Support.Design.XamarinAndroidSdkVersion/build/MonoAndroid81/Xamarin.Android.Support.Design.targets')" />
   <Import Project="$(SolutionDir)/packages/Xamarin.Android.Support.v7.MediaRouter.XamarinAndroidSdkVersion/build/MonoAndroid81/Xamarin.Android.Support.v7.MediaRouter.targets" Condition="Exists('$(SolutionDir)/packages/Xamarin.Android.Support.v7.MediaRouter.XamarinAndroidSdkVersion/build/MonoAndroid81/Xamarin.Android.Support.v7.MediaRouter.targets')" />
   <Import Project="$(SolutionDir)/packages/Xamarin.Forms.XamarinFormsSdk/build/netstandard1.0/Xamarin.Forms.targets" Condition="Exists('$(SolutionDir)/packages/Xamarin.Forms.XamarinFormsSdk/build/netstandard1.0/Xamarin.Forms.targets')" />
-  <Import Project="../.nuget/NuGet.targets" />
```

Comment:
Not sure why this is needed - causes builds of the created project to fail as it tried to find NuGet.exe in the project